tez-dag/src/main/java/org/apache/tez/dag/app/RecoveryParser.java

```diff
@@ -741,12 +743,10 @@ public DAGRecoveryData parseRecoveryData() throws IOException {
             + lastRecoveryFile);
         break;
       }
-      FileStatus fileStatus = recoveryFS.getFileStatus(dagRecoveryFile);
       lastRecoveryFile = dagRecoveryFile;
-      LOG.info("Trying to recover dag from recovery file"
-          + ", dagId=" + lastInProgressDAG.toString()
-          + ", dagRecoveryFile=" + dagRecoveryFile
-          + ", len=" + fileStatus.getLen());
+      LOG.info("Trying to recover dag from recovery file, dagId={}, dagRecoveryFile={}", lastInProgressDAG,
```

Comment:
removed fileStatus.getLen() from the log message, is it intentional?

Reply:
yes, it was shooting an RPC just for log and file length, so removed it

Reply:
we about leaving the info on DEBUG level, but in that case, we can log the full FileStatus, like

if (LOG.isDebugEnabled()) {
    // extra RPC call
    FileStatus fileStatus = recoveryFS.getFileStatus(dagRecoveryFile);
    LOG.debug("Recovery file details: {}", fileStatus);
}